src/extrapolation/extrapolation.jl

```diff
@@ -71,8 +71,8 @@ end
         eflag = tcollect(etpflag, etp)
         xs = inbounds_position(eflag, bounds(itp), x, etp, x)
         g = @inbounds gradient(itp, xs...)
-        skipni = t->skip_flagged_nointerp(itp, t)
-        SVector(extrapolate_gradient.(skipni(eflag), skipni(x), skipni(xs), Tuple(g)))
+        skipni = Base.Fix1(skip_flagged_nointerp, itp)
+        SVector(map(extrapolate_gradient, skipni(eflag), skipni(x), skipni(xs), Tuple(g)))
```

Comment:
Does the map versus broadcast make a difference here?

Reply:
IIUC, `Tuple(g)` itself should have skipped the "nointerp" dimensions, thus `skipni(eflag)`, `skipni(x)`, `skipni(xs)`, `Tuple(g)` should share the same length.
So this change will give the same result.
The only difference is that L75 is still inlinable with `extrapolate_gradient.()` even if `ndims(itp) > 32`, (as it's `ntuple`-based).
But i think there's no practical usage for such a high-dimensional problem, so `map` is ok here.